packages/cli/tests/e2e/bot/FuncHostedNotificationHappyPathCommon.ts

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @author Siglud
+ */
+
+import * as fs from "fs-extra";
+import * as path from "path";
+import { BotValidator } from "../../commonlib";
+import {
+  cleanUp,
+  execAsync,
+  execAsyncWithRetry,
+  getSubscriptionId,
+  getTestFolder,
+  getUniqueAppName,
+  readContextMultiEnv,
+} from "../commonUtils";
+import { environmentManager } from "@microsoft/teamsfx-core";
+
+import { Runtime } from "../../commonlib/constants";
+
+export async function happyPathTest(runtime: Runtime) {
+  const testFolder = getTestFolder();
+  const appName = getUniqueAppName();
+  const subscription = getSubscriptionId();
+  const projectPath = path.resolve(testFolder, appName);
+  const envName = environmentManager.getDefaultEnvName();
+
+  const env = Object.assign({}, process.env);
+  env["TEAMSFX_CONFIG_UNIFY"] = "true";
```

Comment:
Since this flag maybe removed after GA, do we still need to set them for test cases?

Reply:
Yes, I think we can remove those old flag now